src/wirecraft/server/database/models.py

```diff
@@ -18,3 +22,32 @@ class Device(SQLModel, table=True):
     type: str
     x: int
     y: int
+    id_level: int | None = Field(default=None, foreign_key="level.id")  # Lowercase
```

Comment:
you don't have to tell # Lowercas everywhere. SQLModel use the class name lowered as the table name, so it is in lowercase